zipline/pipeline/domain.py

```diff
@@ -100,27 +123,53 @@ class EquityCalendarDomain(Domain):
         ISO-3166 two-letter country code of the domain
     calendar_name : str
         Name of the calendar, to be looked by by trading_calendar.get_calendar.
+    data_query_offset : datetime.timedelta
+         The offset from market open when data should no longer be considered
+         available for a session.
     """
     @expect_types(
         country_code=str,
         calendar_name=str,
         __funcname='EquityCountryDomain',
     )
-    def __init__(self, country_code, calendar_name):
+    def __init__(self,
+                 country_code,
+                 calendar_name,
+                 data_query_offset=-datetime.timedelta(minutes=45)):
```

Comment:
Can we add a test somewhere that passes this with a non-default value and verifies that it affects results?